app/src/main/res/values/strings.xml

```diff
@@ -23,31 +23,31 @@
     <string name="send_kin">Send Kin</string>
     <string name="buying">Buying</string>
     <string name="share_code_to">Share code to</string>
-    <string name="dialog_no_good_left_message">Please try again later</string>
+    <string name="dialog_no_good_left_message">Please check back later</string>
     <string name="dialog_no_good_left_title">Oops! Someone grabbed the last one... for now</string>
     <string name="dialog_coupon_redeem_failed_title">Oops! Transaction incomplete</string>
-    <string name="dialog_coupon_redeem_failed_message">Sorry! We were unable to retrieve your code. Please contact support to resolve this issue.</string>
+    <string name="dialog_coupon_redeem_failed_message">Sorry! We’re unable to retrieve your code. Please contact support.</string>
```

Comment:
@natanrolnik don't see this on the iOS PR

Reply:
So I searched for it, and on iOS we have a different message when it fails to retrieve the code:

Your reflected balance may not be correct. Please contact support to resolve this issue.